ctapipe/containers.py

```diff
@@ -47,16 +47,25 @@
     "TelEventIndexContainer",
     "ImageParametersContainer",
     "SimulatedShowerDistribution",
+    "EventType",
 ]
 
 
+class EventType(enum.Enum):
+    """ note these will evolve once CTA has standardized them """
+
+    data = 0  # standard physics event
+    calibration = 1  # event from calibration system
```

Comment:
Can you directly add values for `Dark`, `Pedestal` and `LaserCalibration`? these are in simtel and are available in the new eventio version

Reply:
yes, that would be great.  For old data, it's still just called "calibration" right?  Are the new emitted by pyeventio also upper-case like "LaserCalibration"? 


I'm also trying to figure out if we can understand if there are CTA-approved integers to use for these.

Reply:
There is an integer type now.

Reply:
And this works for all data that has those events, not only for newly processed data. This ID is there since a long time, iot what just not passed on by the `eventio.SimTelFile`.

Reply:
@param type  Dark (0), pedestal (1), in-lid LED (2) or laser/LED (3+) data.

Reply:
So that's a second integer? E.g. a sub-type of the event_Type?

Reply:
Right now I get it as `array_event["type"]` in SimTelEventSource, and that seems to return a string. Has that changed?

Reply:
No, but there is an additional `array_event['calibration_type']` for calibration events

Reply:
we could for now use 10+calibration_type for the event_type of calibration events, so then you get 0=data, 10,11,12... , rather than adding another field, though another field is also ok

Reply:
You could simply enumerate the types and we can fill it appropriately in the source.

Reply:
I think if we wanted to support the types known to simtel, this should be:

```
data = 0
dark = 1
pedestal = 2
internal_led = 3
external_led_or_laser = 4  # not sure about this one, but I guess external LED / Laser is used for flat fielding? So maybe just call it flatfielding?

Reply:
I'll map these to the ones in the latest (internal so far) document from CTA, where there are something like 10 event/trigger types defined. They are a bit oddly numbered, so may change, but at least it's closer to what will be the standard.

Reply:
can you tell me what are the possible values of `array_event['calibration_type']`?

Reply:
I gave them above:
from io_hess.h:
```
@param type Dark (0), pedestal (1), in-lid LED (2) or laser/LED (3+) data.
```

Reply:
Ah, I didn't realize that was the actual values. Thanks

Reply:
Don't know though what the `3+` is supposed to mean.